civis/base.py

```diff
@@ -1,7 +1,10 @@
+from __future__ import absolute_import
```

Comment:
:man_shrugging: I would hope tests cover this, but I don't know where if anywhere that we're doing float division in here?